hudi-client/hudi-spark-client/src/main/java/org/apache/hudi/index/bloom/SparkHoodieBloomIndex.java

```diff
@@ -252,11 +252,9 @@ public boolean isImplicitWithStorage() {
    * Make sure the parallelism is atleast the groupby parallelism for tagging location
    */
   JavaPairRDD<HoodieKey, HoodieRecordLocation> findMatchingFilesForRecordKeys(
-      final Map<String, List<BloomIndexFileInfo>> partitionToFileIndexInfo,
-      JavaPairRDD<String, String> partitionRecordKeyPairRDD, int shuffleParallelism, HoodieTable hoodieTable,
+      JavaRDD<Tuple2<String, HoodieKey>> fileComparisonsRDD,
```

Comment:
@shenh062326 FYI, related to your Tuple2 refactoring PR 😉 .

Reply:
I am removing the scala dependency from hudl-client-common, it seems no relationship to with this PR, since it depends on scala.Tuple2 in hudi-spark-client.